pages/item/_itemId.vue

```diff
@@ -38,6 +68,33 @@ export default Vue.extend({
     ).data.Items as BaseItemDto[];
 
     this.item = Item[0];
+
+    if (this.item.Type === 'Series') {
+      const seasons = (
+        await this.$tvShowsApi.getSeasons({
+          userId: this.$auth.user.Id,
+          seriesId: this.item.Id || ''
+        })
+      ).data.Items as BaseItemDto[];
```

Comment:
Before we go too far with the BaseItemDto mess, it might be good to figure out what should be split out into media specific components to avoid huge switch statements all over the codebase.

Reply:
We can also add mixins for stuff that needs to be reused.

Reply:
tv shows section has been moved into a seperate component